teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -334,7 +334,8 @@ private void initialise() {
                 config.validatorClient().getValidatorConfig().getValidatorExternalSignerUrl()),
             asyncRunner,
             metricsSystem,
-            dataDirLayout);
+            dataDirLayout,
+            config.validatorClient().getValidatorConfig().allowNoLoadedKeys());
```

Comment:
`config.validatorClient().getValidatorConfig()` is being used a couple of times in this method. I think extracting it into a variable can improve readability.